### PR TITLE
Tests for property tile

### PR DESCRIPTION
**Pull request by satyasekharcvb**
### What does this PR do?
Test for the property tile component

[X ] Tests for the proposed changes have been added/updated.
[ X] Code linting and formatting was performed.

### Functionality Before

NA

### Functionality After

NA

### CODE REVIEW

File: force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        const element = createElement('c-property-tile', {
+            is: PropertyTile
+        });
+        element.property = PROPERTY_DETAILS;
+        document.body.appendChild(element);
+
+        const headerEl = element.shadowRoot.querySelector('.truncate');
+        expect(headerEl.textContent).toBe(PROPERTY_DETAILS.City__c);
+
+        const pElement = element.shadowRoot.querySelector('p');
```
Comment (albarivas) [inline force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js:30]: We're calling element vars whateverEl in all the repos - would you mind to follow that rule for consistency?

File: force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        // Mock handler for child event
+        const handler = jest.fn();
+        element.addEventListener('selected', handler);
+
+        const anchorEl = element.shadowRoot.querySelector('a');
+        anchorEl.click();
+
+        return Promise.resolve().then(() => {
+            // Validate if event got fired
+            expect(handler).toHaveBeenCalled();
```
Comment (albarivas) [inline force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js:70]: Can you check that the event carries the `property.Id` in its `detail` property?

File: force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js]: Changes are done to maintain naming consistency

File: force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    });
+
+    it('fires the property selected event on click', () => {
+        const element = createElement('c-property-tile', {
+            is: PropertyTile
+        });
+        element.property = PROPERTY_DETAILS;
+        document.body.appendChild(element);
+
+        // Mock handler for child event
```
[the remark this answers is not in the document]
Comment (satyasekharcvb) [reply force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js:62]: done

File: force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js (hunk 2 above)
Comment (satyasekharcvb) [reply force-app/main/default/lwc/propertyTile/__tests__/propertyTile.test.js:71]: Check implemented

**Pull request merged**